packages/plugin-core/src/commands/pods/BaseExportPodCommand.ts

```diff
@@ -206,12 +206,17 @@ export abstract class BaseExportPodCommand<
               if (typeof noteProp === "string") {
                 throw new Error("Invalid Payload Type in Pod Note Export");
               } else if (pod.exportNote) {
-                const result = await pod.exportNote(noteProp);
-                await this.onExportComplete({
-                  exportReturnValue: result,
-                  payload: noteProp,
-                  config: opts.config,
-                });
+                try {
+                  const result = await pod.exportNote(noteProp);
+                  await this.onExportComplete({
+                    exportReturnValue: result,
+                    payload: noteProp,
+                    config: opts.config,
+                  });
+                } catch (err) {
```

Comment:
Is this try catch actually necessary? I believe it should get logged as part of `BaseCommand.run()`

Reply:
i found that the error wasn't being propagated with the right message. we should actually do a try/catch at this entire level to get the clean error message but will leave that for another pr